config/config.php

```diff
@@ -1,33 +1,23 @@
 <?php
 
-use Zend\Stdlib\ArrayUtils;
-use Zend\Stdlib\Glob;
+use Zend\ConfigAggregator\ConfigAggregator;
+use Zend\ConfigAggregator\PhpFileProvider;
 
-/**
- * Configuration files are loaded in a specific order. First ``global.php``, then ``*.global.php``.
- * then ``local.php`` and finally ``*.local.php``. This way local settings overwrite global settings.
- *
- * The configuration can be cached. This can be done by setting ``config_cache_enabled`` to ``true``.
- *
- * Obviously, if you use closures in your config you can't cache it.
- */
+$aggregator = new ConfigAggregator(
+    [
+        // Load module config
+        new PhpFileProvider('src/App/Config/{{,*.}global}.php'),
```

Comment:
This should be something like `App\ConfigProvider::class` (to be created) per @weierophinney's suggestion above, instead of looking for individual config files.